llvm/lib/Transforms/Vectorize/SLPVectorizer.cpp

```diff
@@ -113,6 +113,10 @@ static cl::opt<bool>
     RunSLPVectorization("vectorize-slp", cl::init(true), cl::Hidden,
                         cl::desc("Run the SLP vectorization passes"));
 
+static cl::opt<bool>
+    RunSLPReVectorization("revectorize-slp", cl::init(false), cl::Hidden,
+                          cl::desc("Run the SLP revectorization passes"));
```

Comment:
I rather doubt it runs a new pass. Better to say something like "Enable vectorization for wider vector utilization" or something like this.